lib/pavilion/plugins/commands/show.py

```diff
@@ -390,69 +415,126 @@ def _functions_cmd(self, _, args):
                 title="Available Expression Functions"
             )
 
-    @show_cmd('host')
-    def _hosts_cmd(self, pav_cfg, args):
-        """List all known host files."""
+    def show_vars(self, pav_cfg, config, conf_type):
+
+        file = resolver.TestConfigResolver(pav_cfg)._find_config(conf_type,
+                                                                 config)
+        with file.open() as config_file:
+            config_data = file_format.TestConfigLoader().load(config_file)
+
+        data = []
+        col_names = ['Variables', ' ', 'Type', 'Value']
+
+        name = config
+        path = file
+        config = config_data
+
+        for var in config['variables'].keys():
+            var_out = var
+            index = 0
+            for subvar in config['variables'][var]:
+                if type(subvar) is yaml_config.elements.ConfigDict:
+                    type_out = 'dict'
+                    index_out = index
+                    for key, val in subvar.items():
+                        data.append({
+                            'Variables': var_out,
+                            ' ': index_out,
+                            'Type': type_out,
+                            'Value': key + ": " + val,
+                        })
+                        var_out = ''
+                        type_out = ''
+                        index_out = ' '
+                    index += 1
+                else:
+                    data.append({
+                        'Variables': var_out,
+                        ' ': index,
+                        'Type': type(subvar).__name__,
+                        'Value': subvar,
+                    })
+                    var_out = ''
+                    index += 1
 
-        hosts = []
+        output.fprint("Showing Variables for {} {} config at {}: "
+                      .format(name, conf_type.strip('s'), path))
+
+        if data:
+            output.draw_table(
+                self.outfile,
+                field_info={},
+                fields=col_names,
+                rows=data
+            )
+
+    def show_table(self, pav_cfg, args, conf_type):
+
+        configs = resolver.TestConfigResolver(pav_cfg).find_all_configs(conf_type)
+
+        data = []
```

Comment:
A one column table is kind of sad. Might as well drop verbose and always print the path.

Instead, you do need to add an option (like with `pav show tests` to print any errors encountered when loading the config.